en/references/exercise-description/index.md

```diff
@@ -249,13 +249,13 @@ You can easily render code fragments in a monospaced font with syntax highlighti
 When using HTML, wrap your code with `<code>` and `</code>`:
 
 ```html
-In your solution, you can use variable <code>someVariable</code>.
+In your solution, you can use variable <code>some_variable</code>.
 ```
 
-In Markdown, wrap your code with backticks (\`):
+In Markdown, wrap your code with backticks (\`). You can even add the programming language's syntax highlighting with `{:.language-python}` for example.
```

Comment:
Same remarks as the Dutch version: we should probably also do this in the HTML example then.

Reply:
I don't know if this is documented yet, but you could write
```
<div markdown="1">
`print(input())`{:.language-python}
</div>
```
in an HTML document.

Reply:
Ah, this is not true anymore apparently.